src/app/submission/import-external/submission-import-external.component.spec.ts

```diff
@@ -177,6 +184,326 @@ describe('SubmissionImportExternalComponent test suite', () => {
     });
   });
 
+  describe('handle BE response for search query', () => {
```

Comment:
What does "BE" stand for here?  (After more thought, I realize you probably mean "backend")

Could we change this text to just say "Handle response for search query"?  Or "handle backend response..."?  I think abbreviations should be avoided unless they are very obvious (and if I was confused by "BE" then others likely would be as well)

Reply:
Hi @tdonohue , description has been updated